src/routes/posts.js

```diff
@@ -33,7 +33,36 @@ const createPost = async (req, res, next) => {
     }
 }
 
+// GET ALL ARTICLES
+
+const fetchPosts = async (req, res, next) => {
+    try {
+        const allArticles = await db.query('SELECT * FROM posts')
+
+        const articles = allArticles.rows
+        console.log(articles)
+
+        res.status(201).json({
+            status: 'success',
+            // data: articles.map((article) => {
+            // id : article.id,
+            // userId : article.userId,
+            // createdAt : article.createdAt,
+            // title : article.title,
+            // content : article.content,
+            // image : article.image,
+            // published : article.published
+            // createdAt : article.createdAt
+            // }),
```

Comment:
```suggestion
             data: articles.map((article) => ({
             id : article.id,
             userId : article.userId,
             createdAt : article.createdAt,
             title : article.title,
             content : article.content,
             image : article.image,
             published : article.published
             createdAt : article.createdAt
             })),
```

yea so when the code has no `return` and you're returning an object, it should have parenthesis before the object

like 
```js
const func = () => ({
  id: 1,
  name: 'Carl'
})
```
**NOT**

```js
const func = () => {
 .id: 1,
  name: 'Carl'
}
```

Reply:
Thanks so much @jocrah , it worked perfectly, and i have pushed my final code for review